inc/notificationtarget.class.php

```diff
@@ -1536,6 +1536,21 @@ function getUserByField($field, $search_in_object = false) {
    }
 
    /**
+    * Add new recipient with lang to current recipients array
+    *
+    * @param array $data Data (users_id, lang[, field used for notification])
+    *
+    * @deprecated Use NotificationTarget::addToRecipientsList()
+    *
```

Comment:
it's better to add since 9.2 to know in which version we can deleted it
@deprecated since 9.2